### PR TITLE
Adding aria-label to buttons for increased accessibility

### PR DESCRIPTION
**Pull request by gmerritt**
This set of changes addresses several occurrences of a "Button missing a text alternative" condition related to WCAG 2.1's item "4.1.2 Name, Role, Value". An explicit "aria-label" value was added to a button in each case.

### CODE REVIEW

File: apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.ejs [hunk 1]
```diff
@@ -44,14 +44,14 @@
                                 <th id="earliest-access-date">
                                     Earliest Access Date
                                     <a role="button" class="btn btn-xs btn-light" data-placement="auto" data-trigger="focus" data-toggle="popover" data-html="true" title="Order By" tabindex="0"
-                                        data-content="<%= include('popoverStartDate'); %>">
+                                        data-content="<%= include('popoverStartDate'); %>" aria-label="Help on Earliest Access Date">
```
Comment (nwalters512) [inline apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.ejs:47]: As long as you're here: the `title` here doesn't seem relevant to this column. Could you update it to "Earliest Access Date"? Ditto for "Latest Access Date" below.

"Help on ..." reads a little awkwardly to me. Consider "Information about Earliest Access Date" instead; how do you feel about that?

**Review by nwalters512 — COMMENTED** [review]
I won't block on this comment, but let's chat a bit about the right text for `aria-label` on `instructorCourseAdminInstances.ejs` first!

File: apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.ejs (hunk 1 above)
Comment (gmerritt) [reply apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.ejs]: Totally fine, all good! Making changes....

Thanks!

Comment (gmerritt) [reply apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.ejs]: Ohhh yes right that title! Order by! Yes, I had noticed that on the web, and then got too distracted by the second part of your comment. Adjusting that title now!

**Review by nwalters512 — APPROVED** [review]
